target_chains/cosmwasm/contracts/pyth/src/state.rs

```diff
@@ -68,10 +68,10 @@ pub struct ConfigInfo {
 #[derive(Serialize, Deserialize, Clone, Debug, Default, PartialEq, Eq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub struct PriceInfo {
```

Comment:
I think we can remove the PriceInfo entirely. We had it before but these extra information here are not useful for people and add storage cost. We can only store PriceFeed.